client/assets/images/README.md

```diff
@@ -0,0 +1,8 @@
+# Images
+
+Images from this folder are managed by Webpack, and can be imported like this:
```

Comment:
webpack is spelled in all lowercase, kind of opposite of the capital-P-dangit rule 🙂

Reply:
I didn't know about that, I'll change it.